client/src/handlers/add-item.js

```diff
@@ -2,15 +2,30 @@ const todoInput = document.querySelector('.todo-input');
 const todoList = document.querySelector('.todo-list');
 
 export const addItem = (event) => {
+  // event.preventDefault();
+  // const todoDiv = document.createElement('div');
+  // todoDiv.classList.add('todo');
+  // const newItem = document.createElement('li');
+  // newItem.innerText = todoInput.value;
+  // newItem.classList.add('todo-item');
+  // todoDiv.append(newItem);
+  // todoList.append(newItem);
+
+  // //  clear todo input value
+  // todoInput.value = '';
+
   event.preventDefault();
-  const todoDiv = document.createElement('div');
-  todoDiv.classList.add('todo');
-  const newItem = document.createElement('li');
-  newItem.innerText = todoInput.value;
-  newItem.classList.add('todo-item');
-  todoDiv.append(newItem);
-  todoList.append(newItem);
+  const itemName = todoInput.value;
+  const newLi = document.createElement('li');
+  const cb = document.createElement( "input" );
+  cb.type = "checkbox";
+  cb.id = 'chk-' + itemName;
+  cb.checked = false;
+  newLi.appendChild(cb);   // Append the checkbox to the <li>
+  var newItem = document.createTextNode(itemName);  // Create the text node after the the checkbox
```

Comment:
maybe: 
 const newItem = document.createTextNode(' ' + itemName);  // Create the text node after the the checkbox

Reply:
it is probably better to use CSS margin-right instead of ' '

Reply:
Yes good suggestion. Will do it